README.md

```diff
@@ -135,9 +135,6 @@ Contributors are welcome see the [Contributing](CONTRIBUTING.md) guidelines for
 
 ### Optional components
 
-<details>
-  <summary>Click to expand!</summary>
-
 A list of optional component that are not included in HELICS but are optionally used by the library
```

Comment:
may be at some point we should just move this to a separate md file and link it in the readme.

Reply:
They seem like they serve a similar purpose as the list of libraries on https://github.com/GMLC-TDC/HELICS/blob/main/CONTRIBUTORS.md#used-libraries-or-code so maybe the two lists could get merged later?

Reply:
the distinction is that one list is required and one is optional,  which has some legal distinction particularly with zmq.  But we may be able to put an optional component section in the contributors list.

Reply:
I think both lists are just explaining how the libraries get used so separate lists don't make much difference. If we want to make an optional component section for libraries, we'd want to move Boost to that list as well.

It does feel like the list is less for users/developers and more for the convenience of whoever is reviewing software release requests.

Reply:
I'll leave this PR as-is for now, and that can be a later change.

Reply:
Agreed lets clean this up by the next release, probably move this list to a section of the contributors md file.